themes/bootstrap3/templates/RecordDriver/DefaultRecord/list-entry.phtml

```diff
@@ -33,36 +33,37 @@
     <?php endif; ?>
     <div class="media-body">
       <div class="result-body">
-        <div class="resultItemLine1">
-          <?php $missing = $this->driver instanceof \VuFind\RecordDriver\Missing; ?>
-          <?php if (!$missing): ?><a href="<?=$this->escapeHtmlAttr($this->recordLinker()->getUrl($this->driver))?>" class="getFull" data-view="<?=$this->params->getOptions()->getListViewOption() ?>"><?php endif; ?>
-            <span class="title"><?=$this->record($this->driver)->getTitleHtml()?></span>
-          <?php if (!$missing): ?></a><?php endif; ?>
-          <?php foreach ($this->driver->tryMethod('getTitlesAltScript', [], []) as $altTitle): ?>
-            <div class="title-alt">
-              <?=$this->escapeHtml($altTitle)?>
-            </div>
-          <?php endforeach; ?>
-        </div>
+        <header>
+          <h2>
+            <?php $missing = $this->driver instanceof \VuFind\RecordDriver\Missing; ?>
+            <?php if (!$missing): ?><a href="<?=$this->escapeHtmlAttr($this->recordLinker()->getUrl($this->driver))?>" class="getFull" data-view="<?=$this->params->getOptions()->getListViewOption() ?>"><?php endif; ?>
+              <span class="title"><?=$this->record($this->driver)->getTitleHtml()?></span>
+            <?php if (!$missing): ?></a><?php endif; ?>
+            <?php foreach ($this->driver->tryMethod('getTitlesAltScript', [], []) as $altTitle): ?>
+              <div class="title-alt">
+                <?=$this->escapeHtml($altTitle)?>
+              </div>
+            <?php endforeach; ?>
```

Comment:
Do we want all of the alts inside of the h2?

Reply:
Well, the title should be in the h2 and these conditions belong to it, right?

Reply:
The variable/class naming may be misleading here. Based on a quick glance at the code, I thought these represented "alternate titles" in which case I would agree with @crhallberg that they do not belong inside the h2. However, I see on closer examination that these are actually alternate script forms for the primary title -- e.g. Chinese characters to accompany a Romanized version of the same text. In that scenario, I agree with @elsenhans that it is appropriate to include them in the heading. So I think this is fine as it is.